packages/saber/lib/plugins/transformer-markdown.js

```diff
@@ -80,10 +80,10 @@ function transformMarkdown(api, page) {
       name: 'headings',
       resolve: require.resolve('../markdown/headings-plugin'),
       options: {
+        ...markdown.headings,
```

Comment:
Bit dangerous, what if `headings` is undefined? Also validateConfig will throw an error on start if you use this option in `saber-config`.

Reply:
`undefined` is fine.

Reply:
always thought it would throw an error :eyes: